generated-platform-project/quarkus-camel/bom/pom.xml

```diff
@@ -647,17 +647,17 @@
       <dependency>
         <groupId>io.dropwizard.metrics</groupId>
         <artifactId>metrics-healthchecks</artifactId>
-        <version>4.1.19</version>
+        <version>4.0.1</version>
```

Comment:
This change (and other dropwizard, org.graalvm.js and apache.curator) indicate that your BOM includes other artifacts that originate from different releases of the same project. I.e. they were previously aligned by the platform generator but now they are not.

Reply:
This is a bit strange.

For dropwizard and Camel Quarkus 2.4.0, I'd expect the version 4.1.25, defined in `org.apache.camel:camel-dependencies:3.12.0`

Similarly `org.graalvm.js` should be at `21.2.0` see https://github.com/apache/camel-quarkus/blob/2.4.0/poms/bom/pom.xml#L6084 and https://github.com/apache/camel-quarkus/blob/2.4.0/pom.xml#L83

Reply:
You have two artifacts coming from `org.graalvm.js`: `js` one is `21.2.0` but the `js-scriptengine` is `20.0.0`.
As to the dropwizard, you should check but that's what they actually are. An easy way to verify it is to download https://repo1.maven.org/maven2/org/apache/camel/quarkus/camel-quarkus-bom/2.4.0/camel-quarkus-bom-2.4.0.pom as `pom.xml` and run `mvn help:effective-pom > effective-pom.xml`. You'll see the flattened version of your BOM with all the actual constraints.

Reply:
Going to check that now

Reply:
You are right, the BOM generator works as designed, we need to be more careful what we manage in our BOM. I filed https://github.com/apache/camel-quarkus/issues/3302 for that. 
These insights are super useful for us - thanks a lot for them!